lib/collective.lib.js

```diff
@@ -150,3 +150,9 @@ export const getContributeRoute = collective => {
 export const getSuggestedTags = collective => {
   return collective?.expensesTags?.map(({ tag }) => tag) || [];
 };
+
+/** Given a user and a collective see if the user is an admin of other collective under the same host **/
+export const isUserAdminOfCollectiveUnderSameHost = (user, account) => {
+  const sameHostCollectives = user?.memberOf.filter(obj => obj.collective?.host?.id === account.host.legacyId);
+  return sameHostCollectives?.length > 0;
```

Comment:
Nice helper! Using `some` could be a simple way to avoid iterating on the full list:

```suggestion
  return Boolean(user?.memberOf.some(member => member.collective?.host?.id === account.host.legacyId));
```